src/Constants.js

```diff
@@ -321,6 +352,7 @@ export const ZORAS_DOMAIN = [
     "Zoras-Fountain-Iceberg-Freestanding-PoH",
     "Zoras-Fountain-Bottom-Freestanding-PoH",
     "Zoras-Fountain-Fairy-Reward",
+    "King-Zora-Moves",
```

Comment:
There's no item reward for moving King Zora (unless that was changed in Master Quest?)